cmd/vulcan-cli/cmd/users.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2023 Adevinta
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"slices"
+
+	"github.com/adevinta/vulcan-api/cmd/vulcan-cli/cli"
+	"github.com/spf13/cobra"
+)
```

Comment:
```suggestion
import (
	"fmt"
	"os"
	"slices"

	"github.com/spf13/cobra"

	"github.com/adevinta/vulcan-api/cmd/vulcan-cli/cli"
)
```

Organize imports in 3 blocks:

1. Standard library
2. External modules
3. Main module

Reply:
Absolutely my bad.